examples/README.md

```diff
@@ -0,0 +1,6 @@
+# stac-fastapi-elasticsearch (sfes) deployment examples 
+
+## Deploy sfes from pip in docker
+
+```$ cd pip_docker```
+```$ docker-compose up```
```

Comment:
```suggestion
```shell
cd pip_docker
docker-compose up
```

The suggestion isn't perfect because its hard to put a code fence in a code fence, but hopefully you get the idea.

Reply:
Looks better now. Thanks!